### PR TITLE
Move UUri invariants to UAttributes specification

### PR DESCRIPTION
**Pull request by sophokles73**
The invariants for UUri's used in a certain context have been moved
from the UUri to the UAttributes specification where they are now
defined in the proper (message type) context.

### CODE REVIEW

File: basics/uattributes.adoc [hunk 1]
```diff
@@ -102,6 +102,7 @@ The following table defines attributes that are used for publish messages in add
 |yes
 |A link:uri.adoc[UUri] representing the topic that this message is published to.
 
+* The UUri's `resource_id` *MUST* be set to a value in range `[0x8000, 0xFFFF[`.
```
Comment (stevenhartley) [inline basics/uattributes.adoc:105]:
```suggestion
* The UUri's `resource_id` *MUST* be set to a value in range `[0x8000, 0xFFFF]`.
```

File: basics/uattributes.adoc [hunk 2]
```diff
@@ -121,10 +122,14 @@ The following table defines attributes that are used for notification messages i
 |yes
 |A link:uri.adoc[UUri] representing the component that this notification originates from.
 
+* The UUri's `resource_id` *MUST* be set to a value in range `[0x8000, 0xFFFF[`.
```
Comment (stevenhartley) [inline basics/uattributes.adoc:125]:
```suggestion
* The UUri's `resource_id` *MUST* be set to a value in range `[0x8000, 0xFFFF]`.
```

File: basics/uattributes.adoc [hunk 3]
```diff
@@ -144,10 +149,14 @@ The following table defines attributes that are used for RPC request messages in
 |yes
 |The link:uri.adoc[UUri] that the service consumer expects to receive the response message at.
 
+* The UUri's `resource_id` *MUST* be set to `0`.
+
 |`sink`
 |yes
 |A link:uri.adoc[UUri] identifying the service provider's method to invoke.
 
+* The UUri's `resource_id` *MUST* be set to a value in range `]0, 0x8000[`.
```
Comment (stevenhartley) [inline basics/uattributes.adoc:158]:
```suggestion
* The UUri's `resource_id` *MUST* be set to a value in range `[1, 0x7FFFF]`.
```

**Review by stevenhartley — CHANGES_REQUESTED** [review]
Not sure I understood the bracket notation, I think it is simpler to refer to them as I showed

File: basics/uattributes.adoc (hunk 1 above)
Comment (sophokles73) [reply basics/uattributes.adoc:105]: My bad, this is supposed to be the (open) interval `[0x8000, 0xFFFF)`, using mathematical notation ...

**Review by stevenhartley — APPROVED** [review]
I added fix for proto links that I recently broke and approve the change

**Pull request merged**